### ISSUE TITLE
Knowledge Graph - Should have

### ISSUE THREAD

**Issue by KoSchoemaker** [stijnli/information-visualization-project#5]
Should be picked up after #2 

- [ ] Contextmenu met rechtermuisknop. perhaps could also be added to the on-hover (more likely will be seen there). should be functionality of adding to selection and opening in spotify

**Comment by stijnli** [comment]
Maybe this can become a nice to have? As one can add the song by right clicking it and then open the song on spotify by clicking the song within the selected songs list? @KoSchoemaker @vitusch

**Comment by KoSchoemaker** [comment]
add/remove from selection already possible right mouse, open spotify from left menu

can be closed
